### DISCUSSION TITLE
怎么手动下载 shibing624/mengzi-t5-base-chinese-correction 这个数据集啊

### DISCUSSION THREAD

**Discussion by sovereign-stone** [shibing624/pycorrector#487] [General]
大佬们，怎么直接通过url下载 shibing624/mengzi-t5-base-chinese-correction 这个数据集呀，用 huggingface_hub，transformers，datasets 一直报错，加了代理也是报错，有没有大佬分享个直接下载的链接啊

**Comment by shibing624** [comment]
https://hf-mirror.com/

**Reply by sovereign-stone** [reply]
好的，收到！
